backend/x11/input_device.c

```diff
@@ -2,11 +2,7 @@
 
 #include <wlr/config.h>
 
-#ifdef __linux__
```

Comment:
Hmm, what's the story about these? Are `linux/*` includes available on FreeBSD now?

Reply:
devel/evdev-proto currently provides `<linux/input.h>`, `<linux/input-event-codes.h>`, `<linux/uinput.h>`. Those files are usually newer than what's shipped in FreeBSD base.

@wulf7, do you know when consumers should use `<dev/evdev/*>` and when `<linux/*>`?

Reply:
The rule is simple: always use <linux/*> except you are writing something FreeBSD-only like kernel drivers

Reply:
Okay, thanks!